db/document.go

```diff
@@ -817,14 +817,6 @@ func (doc *Document) IsChannelRemoval(revID string) (bodyBytes []byte, history R
 		return nil, nil, nil, false, false, nil
 	}
 
-	// Construct removal body
```

Comment:
Looks like bodyBytes is no longer used by IsChannelRemoval.  Can it be removed from the return values?

Reply:
Yeah, this can be removed.